.github/workflows/_en-gb-localise.yml

```diff
@@ -0,0 +1,64 @@
+name: Check existens of En_GBLocalise in translations
+
+on:
+  pull_request:
+    branches:
+      - main
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+
+jobs:
+  engb-localise:
+    runs-on: ubuntu-latest
+    
+    permissions:
```

Comment:
@wojsmol did we need the permissions?

We have not set permissions in any GHA.